source/technical/reservations.rst

```diff
@@ -156,6 +158,16 @@ Instead of specifying the node type, you may also reserve a specific node by pro
 
    blazar lease-create --physical-reservation min=1,max=1,resource_properties='["=", "$uid", "c9f98cc9-25e9-424e-8a89-002989054ec2"]' --start-date "2015-06-17 16:00" --end-date "2015-06-17 18:00" my-custom-lease
 
+.. _disable-blazar-notification:
+.. attention:: 
+   To specify a ``before_end`` action, simply add ``before_end=<action_type>`` to ``physical-reservation`` parameter. For example:
+   
+   .. code-block:: bash
+
+      blazar lease-create --physical-reservation min=1,max=1,resource_properties='["=", "$uid", "c9f98cc9-25e9-424e-8a89-002989054ec2"]',before_end=email --start-date "2015-06-17 16:00" --end-date "2015-06-17 18:00" my-custom-lease
+      
+   The default ``before_end`` action is ``email``. To disable the email notification, set ``before_end=''``.
```

Comment:
👍

Reply:
The text doesn't make it clear what before_end actions are allowed.